client/client-lib/src/api.rs

```diff
@@ -27,7 +27,7 @@ pub trait LnFederationApi {
         &self,
         payment_hash: Sha256Hash,
     ) -> FederationResult<IncomingContractOffer>;
-    async fn fetch_consensus_clock_time(&self) -> FederationResult<SystemTime>;
+    async fn fetch_consensus_clock_time(&self) -> FederationResult<u64>;
```

Comment:
Not sure if that is an improvement, more specific types are typically better, what was the reason here?